react-multiple-components/my-app/src/Buttons.tsx

```diff
@@ -0,0 +1,33 @@
+import './Numbers.css';
+
+type Props = {
+  text?: string;
+  length?: number;
+  index?: number;
+  onClick?:
+    | (() => void)
+    | ((event: React.MouseEvent<HTMLButtonElement>) => void);
+};
+
+export function Buttons({ length, index, onClick, text }: Props) {
```

Comment:
You combined both styles of button in one component. That makes this component harder to maintain. I think you're better off with two separate components. Components are easy to create, so don't worry about creating extra components.